lib/driverProviders/driverProvider.ts

```diff
@@ -93,56 +92,48 @@ export abstract class DriverProvider {
    *
    * @param drivers {webdriver.WebDriver[]} The webdriver instances
    */
-  static quitDrivers(provider: DriverProvider, drivers: WebDriver[]): q.Promise<void> {
-    let deferred = q.defer<void>();
-    wdpromise
-        .all(drivers.map((driver: WebDriver) => {
-          return provider.quitDriver(driver);
+  static async quitDrivers(provider: DriverProvider, drivers: WebDriver[]): Promise<void> {
+    return await Promise
+        .all(drivers.map(async (driver: WebDriver) => {
+          await provider.quitDriver(driver);
         }))
-        .then(
-            () => {
-              deferred.resolve();
-            },
-            () => {
-              deferred.resolve();
-            });
-    return deferred.promise;
+        .then(() => {});
```

Comment:
Why do we need this?

Reply:
Previously this satisfied typings where we would return a Promise<void>. Promise.all returns an Promise<void[]>. Instead since we have async / await, we could just await Promise.all to return. Without specifying a return type, this could be Promise<void>.

Fixing this.